airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -194,7 +194,8 @@ def run_query(
             write_disposition = 'WRITE_EMPTY',
             allow_large_results=False,
             udf_config = False,
-            use_legacy_sql=True):
+            use_legacy_sql=True,
+            maximum_billing_tier=1):
```

Comment:
Hi Chris, you are right. I totally missed that. I will change it to None/False and test it. Thanks for pointing that out.